libcontainer/criu_opts.go

```diff
@@ -1,5 +1,14 @@
 package libcontainer
 
+type cg_mode uint32
+
+const (
+	CRIU_CG_MODE_SOFT cg_mode = 3 + iota
+	CRIU_CG_MODE_FULL
+	CRIU_CG_MODE_STRICT
+	CRIU_CG_MODE_DEFAULT
+)
```

Comment:
Hi, Sebastiaan,
Thanks for your suggestions. I have added comment to these constants.

On Sat, Aug 8, 2015 at 9:15 PM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> In libcontainer/criu_opts.go
> <https://github.com/opencontainers/runc/pull/184#discussion_r36583389>:
>
> > @@ -1,5 +1,14 @@
> >  package libcontainer
> >
> > +type cg_mode uint32
> > +
> > +const (
> > +	CRIU_CG_MODE_SOFT cg_mode = 3 + iota
> > +	CRIU_CG_MODE_FULL
> > +	CRIU_CG_MODE_STRICT
> > +	CRIU_CG_MODE_DEFAULT
> > +)
>
> (just passing by); if that needs explanation - would it be useful to add
> that information as a comment in the code?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runc/pull/184/files#r36583389>.
>